README.md

```diff
@@ -78,7 +86,7 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-Click <a href="https://tourmaline-chebakia-a54d98.netlify.app/">Here</a>
+Click <a href="https://www.loom.com/share/bb4e92f5ece64a7db30c2d2df187fc1d">Here</a>
```

Comment:
##### **Live version**

- [ ] It's one of the project requirements you deploy the application using Heroku, Netlify, or GitHub Pages. and make sure that:
1. The project is deployed properly, and the home and detail page works as expected. ✔️ 
2. A link to the deployed project is present in the README. ✔️ 
_ Let's address this 👍🏻